### DISCUSSION TITLE
Deployment fails to install runtime - Error: Exited with 1

### DISCUSSION THREAD

**Discussion by good-idea** [vercel/vercel#4872] [Help] [answered]
I opened a support ticket but thought I'd post here in case anyone else is having the same issue. All of my new deployments are failing on the Installing Build Runtime step:

```

14:31:42.902 | Cloning github.com/good-idea/spinellikilcollin.com (Branch: main, Commit: a3c7780)
-- | --
14:31:43.697 | Cloning completed in 794ms
14:31:43.697 | Analyzing source code...
14:31:43.698 | Warning: Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
14:31:45.291 | Installing build runtime...
14:34:00.610 | Error: Exited with 1
14:34:00.610 | at ChildProcess.<anonymous> (/var/task/sandbox-worker.js:21673:37)
14:34:00.610 | at ChildProcess.emit (events.js:310:20)
14:34:00.610 | at ChildProcess.EventEmitter.emit (domain.js:482:12)
14:34:00.610 | at maybeClose (internal/child_process.js:1021:16)
14:34:00.610 | at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14:34:02.866 | Done with "app/package.json"
```

This is happening on multiple projects, deployed from both the CLI and from github. Re-deploying a deployment that was previously successful yields the same results.

```
Vercel CLI 19.1.1
Next.js v9.4.4
```

now.json
```
{
  "version": 2,
  "builds": [{ "src": "app/package.json", "use": "@now/next" }],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "app/$1",
      "continue": true
    }
  ],
  "build": {
    "env": {
      "SC_DISABLE_SPEEDY": "true",
      "SENTRY_DSN": "@jw-sentry-dsn",
      "KLAVIYO_LIST_ID": "@jw-klaviyo-list-id",
      "KLAVIYO_API_KEY": "@jw-klaviyo-api-key",
      "SANITY_PROJECT_ID": "@jw-sanity-project-id",
      "SANITY_DATASET": "@jw-sanity-dataset",
      "SANITY_AUTH_TOKEN": "@jw-sanity-auth-token",
      "SHOPIFY_SHOP_NAME": "jemmawynne-development",
      "SHOPIFY_STOREFRONT_TOKEN": "@jw-storefront-token"
    }
  }
}
```

next.config.js
```
const webpack = require('webpack')
const dotEnv = require('dotenv')
const bundleAnalyzer = require('@next/bundle-analyzer')
const withSourceMaps = require('@zeit/next-source-maps')

dotEnv.config()

const withBundleAnalyzer = bundleAnalyzer({
  enabled: process.env.ANALYZE === 'true',
})

module.exports = withSourceMaps(
  withBundleAnalyzer({
    env: {
      SANITY_PROJECT_ID: process.env.SANITY_PROJECT_ID,
      SANITY_DATASET: process.env.SANITY_DATASET,
      SANITY_READ_TOKEN: process.env.SANITY_READ_TOKEN,
      SENTRY_DSN: process.env.SENTRY_DSN,
    },
    webpack: (config, { isServer, buildId }) => {
      config.plugins.push(
        new webpack.DefinePlugin({
          'process.env.SENTRY_RELEASE': JSON.stringify(buildId),
        }),
      )

      if (!isServer) {
        config.resolve.alias['@sentry/node'] = '@sentry/browser'
      }

      return config
    },
  }),
)
```

**Comment by good-idea** [comment]
welp, now it works.

**Reply by olamcocoa** [reply]
im also having this issue. Any idea how it fixed itself?

**Reply by andersravn** [reply]
Currently getting it as well. But it's for a new project, so I thought I had done something wrong. Is it a bigger issue?

**Reply by phildeschaine0** [reply]
Me too.  Some sort of outage.  Not reported on their status page, though: https://www.vercel-status.com/

**Reply by olamcocoa** [reply]
i raised a ticket. I am also hosting previews via Gatsby Cloud and thats working fine. So i suppose we have not broken the build

**Reply by jdjam** [reply]
Getting the same issue as well. Works locally with 'vercel dev' but fails the build when I attempt to deploy.

**Reply by omaratta212** [reply]
Glad I'm not alone, hope they fix it soon.

**Reply by localgangster001** [reply]
Got this issue too. From the logs it looks like something is failing on NPM side. Here are some of the lines I isolated:

`121 http fetch GET 404 https://registry.npmjs.org/@vercel%2fnext 48ms
122 http fetch GET 404 https://registry.npmjs.org/@vercel%2fnext 37ms
123 silly fetchPackageMetaData error for @vercel/next@2.6.13 404 Not Found - GET https://registry.npmjs.org/@vercel%2fnext - Not found
124 http fetch GET 404 https://registry.npmjs.org/@vercel%2fgo 101ms
125 http fetch GET 404 https://registry.npmjs.org/@vercel%2fbuild-utils 108ms
126 http fetch GET 404 https://registry.npmjs.org/@vercel%2fnode 109ms
127 http fetch GET 404 https://registry.npmjs.org/@vercel%2fruby 109ms
128 http fetch GET 404 https://registry.npmjs.org/@vercel%2fstatic-build 110ms
129 http fetch GET 404 https://registry.npmjs.org/@vercel%2fpython 127ms
130 http fetch GET 404 https://registry.npmjs.org/@vercel%2fgo 34ms`

**Reply by coupon-mad** [reply]
got the issue too.
```

18:14:31.376 | Cloning completed in 2259ms
-- | --
18:14:31.378 | Analyzing source code...
18:14:35.735 | Installing build runtime...
18:14:52.113 | Error: Exited with 1
18:14:52.113 | at ChildProcess.<anonymous> (/var/task/sandbox-worker.js:21953:21)
18:14:52.113 | at ChildProcess.emit (events.js:310:20)
18:14:52.113 | at ChildProcess.EventEmitter.emit (domain.js:482:12)
18:14:52.113 | at maybeClose (internal/child_process.js:1021:16)
18:14:52.113 | at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
18:14:56.297 | Done with "package.json"

```

**Reply by salnikovp0** [reply]
same issue

**Reply by miyamoto-san** [reply]
Yes, it seems to be an error at the moment

**Comment by HaoXiangL** [comment]
I think it is build runtime error,because in my local i can build my project.

**Comment by Delkathes** [comment, marked as answer]
https://www.vercel-status.com

They identified the issue. Npm related. I also had errors installing some npm packages on local earlier today.

**Reply by HaoXiangL** [reply]
Thanks！

**Reply by coupon-mad** [reply]
Thanks!
